doc/recording.md

```diff
@@ -0,0 +1,369 @@
+ 
+# Install rotter (which will also install "jack-audio-connection-kit" and
+# "jack-audio-connection-kit-example-clients")
+yum install rotter
+```
+
+Install the [raar jackd systemd service instance
+override](../config/systemd/jackd@raar.service.d/override.conf):
+```bash
+# Install the jackd@raar service instance unit override
+mkdir "/etc/systemd/system/jackd@raar.service.d"
+wget -O "/etc/systemd/system/jackd@raar.service.d/override.conf" \
+     https://raw.githubusercontent.com/radiorabe/raar/master/config/systemd/jackd%40raar.service.d/override.conf 
+
+# You might need to adapt the jackd service instance unit override to suite
+# your environment
+vi /etc/systemd/system/jackd@raar.service.d/override.conf
+
+# Reload systemd manager configuration
+systemctl daemon-reload
+```
+
+Install the [raar rotter systemd service instance
+override](../config/systemd/rotter@raar.service.d/override.conf):
+```bash
+# Install the rotter@raar service instance unit override
+mkdir "/etc/systemd/system/rotter@raar.service.d"
+wget -O "/etc/systemd/system/rotter@raar.service.d/override.conf" \
+     https://raw.githubusercontent.com/radiorabe/raar/master/config/systemd/rotter%40raar.service.d/override.conf 
+
+# You might need to adapt the rotter service instance unit override to suite
+# your environment
+vi /etc/systemd/system/rotter@raar.service.d/override.conf
+
+# Reload systemd manager configuration
+systemctl daemon-reload
+```
+
+You can now enable and start the `raar@rotter.service` instance, which will
+also trigger the start of the `raar@jackd.service` instance.
+```bash
+# Enable and start the service unit instance
+systemctl enable raar@rotter.service
+systemctl start raar@rotter.service
+
+# Check the status
+systemctl status raar@rotter.service
+systemctl status raar@jackd.service
+
+# In case of problems, inspect the systemd journal
+journalctl -u raar@rotter.service
+journalctl -u raar@jackd.service
+```
+
+The recordings should appear in the `/var/lib/rotter/raar` directory.
+
+As a final step, install the [RAAR Record Handler](#raar-record-handler).
+
+### Deployment on other distributions
+#### Dedicated system user for recording
+First, a dedicated system user and group will be created under which all
+involved processes will run. The user and group is named `rotter` (you might
+need to adapt the supplementary groups, `audio` and `jackuser`, to match with
+your distribution).
+```bash
+# Create a dedicated audio recording user which belongs to the audio and
+# jackuser groups. The later is required in order to gain real-time priority
+# see also /etc/security/limits.d/95-jack.conf
+useradd --comment "rotter system user" \
+        --home-dir "/var/lib/rotter" \
+        --create-home \
+        --groups audio,jackuser \
+        --password '*' \
+        --system \
+        --shell /sbin/nologin \
+        --user-group \
+        rotter
```

Comment:
I'm not sure if you saw https://github.com/radiorabe/centos-rpm-rotter/issues/1 before writing this.